content/docs/1.45/_index.md

```diff
@@ -69,7 +69,7 @@ See [running a docker all in one image](getting-started#all-in-one).
 - [Evolving Distributed tracing At Uber Engineering](https://eng.uber.com/distributed-tracing/) (blog)
 - [Mastering Distributed Tracing](https://shkuro.com/books/2019-mastering-distributed-tracing/) (book)
 - [OpenTracing Tutorial (Java, Go, Python, Node.js, C#)](https://github.com/yurishkuro/opentracing-tutorial/) (tutorials)
-- [IBM Garage Cloud Native Bootcamp - Distributed Tracing](https://cloudnative101.dev/electives/dist-trace/) (tutorials)
+- [IBM Garage Cloud Native Bootcamp - Distributed Tracing](https://www.ibm.com/cloud/blog/distributed-tracing) (tutorials)
```

Comment:
From what I remember of the original link, this is not an equivalent replacement. The original was an actual course, where this blog post is mostly fluff. If we cannot find the original course I suggest we simply remove this entry.

Reply:
Also, we should make the same change across all versioned docs, not just the latest version.

Reply:
I understand. Thank you very much for the review, I'll make the changes soon

Reply:
I'm not able to find the original course. The links which do look similar are pointing to cloudnative101.dev which is not working.
So I think the best course of action would be to remove the links